etebase_server/fastapi/routers/authentication.py

```diff
@@ -122,16 +123,36 @@ def get_login_user(request: Request, challenge: LoginChallengeIn) -> UserType:
             raise AuthenticationFailed(code="user_not_init", detail="User not properly init")
         return user
     except User.DoesNotExist:
-        raise AuthenticationFailed(code="user_not_found", detail="User not found")
+        return fake_user(username)
 
 
-def get_encryption_key(salt: bytes):
+FAKE_USER_COUNT = 1000
+
+
+def fake_user(username: str) -> UserType:
```

Comment:
It's a good idea, but this is unfortunately not effective.

The returned user id is random every time, so just hit this API twice to check for existence. Additionally, I haven't had the time to analyze the using of static pubkeys/salts but I'm concerned that may have issues too.

Reply:
> The returned user id is random every time, so just hit this API twice to check for existence.

Oh, you are absolutely right, didn't think of that